docs/source/extensions/overview_extensions.rst

```diff
@@ -34,9 +36,21 @@ of the core project:
 Community-maintained extensions
 -------------------------------
 
-The following extensions are developed and maintained by the ASReview community:
+ASReview has support for community-maintained extensions, that enable you to seemlessly integrate 
+your own code with the ASReview framework. These extensions can extend the software with new 
+classifiers, query strategies, balance strategies, and feature extraction techniques.
```

Comment:
Make clear that it can extend subcommands, models, datasets? The classifiers, query strategies, balance strategies, and feature extraction techniques are part of models.